cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala

```diff
@@ -133,11 +133,13 @@ class SchemaIndexTest extends DocumentingTestBase with QueryStatisticsTestSuppor
   @Test def list_indexes() {
     prepareAndTestQuery(
       title = "List indexes",
-      text = "Calling the built-in procedure `db.indexes` will list all indexes, including their names.",
+      text =
+        """Listing all indexes can be done with `SHOW INDEXES`, which will produce a table with these columns:
+          |include::list-indexes-table-columns.asciidoc[]""".stripMargin,
```

Comment:
This link seems to be broken, couldn't see the columns.

Reply:
Maybe it can't be added in the text like this? not sure, these tests don't have the same setup as the security tests I looked at. I guess i'll look into it more then

Reply:
It's weird that it works (kinda) for constraints and not for indexes, the link is correct (copied the filename and pasted over and no changes were made on the row)

Have not done any updates for this yet, and don't really know what to do